### ISSUE TITLE
Does not work on .NET MAUI 8

### ISSUE THREAD

**Issue by xarlammm** [thudugala/Plugin.LocalNotification#491]
I started migrating from xamarin to .net maui 8. Nothing happens when I try to show the notification. Permission is also not asked. If you manually allow notifications to be displayed, it is not displayed. 
All permissions are in the manifest. The MauiProgram is also connected.

Android 14
.NET MAUI 8

![image](https://github.com/thudugala/Plugin.LocalNotification/assets/95414457/6bec0e0b-9d3b-4597-863c-30fc5ad9e673)

**Comment by thudugala** [comment]
@xarlammm please go through https://github.com/thudugala/Plugin.LocalNotification/wiki/1.-Usage-10.0.0--.Net-MAUI

**Comment by xarlammm** [comment]
Hello!
All these stages were completed, followed this documentation

**Comment by thudugala** [comment]
@xarlammm can you attach a sample project please?

**Comment by xarlammm** [comment]
I created a completely new project and followed your documentation. The result is the same. 
[NotificationApp.zip](https://github.com/thudugala/Plugin.LocalNotification/files/14998512/NotificationApp.zip)

**Comment by maheshenator** [comment]
.Net MAUI 8 -  Android 13 and above.
Observation in Emulator with Android 14.  Notification comes if we enable Notification permission MANUALLY via "Settings", could not get Notification request Popup automatically. ( May be some platform level implementation needed? )
I had  POST_NOTIFICATIONS , SCHEDULE_EXACT_ALARM , USE_EXACT_ALARM  these permission enabled in manifest.

Apps should take explicit permission it seems - https://source.android.com/docs/core/display/notification-perm

2.  Scheduled notifications doesn't work,  if application is not Active/Open.  If application is closed, then even active notification gets cleared - May be this should this be a new bug?
 
TestRepo : https://github.com/maheshenator/LocalNotificationsTest
 
```
{
    NotificationId = 11,
    Title = $"Hey, Notification.. .",
    Subtitle = "My App Subject",
    Description = "Notific Desc",
    BadgeNumber = 11,
    Schedule = new NotificationRequestSchedule
    {
        NotifyTime = DateTime.Now.AddSeconds(10),
        NotifyRepeatInterval = TimeSpan.FromSeconds(10),
        RepeatType = NotificationRepeat.TimeInterval
    }
};
LocalNotificationCenter.Current.Show(request);
```

**Comment by erossini** [comment]
@keikepono did you publish your app on Google Play? What is your _AndroidManifest.xml_? I can't manage the notifications in my app for Android (see this [post](https://github.com/thudugala/Plugin.LocalNotification/issues/503)).

Thank you for any suggestions.
Enrico

**Comment by ivmirx** [comment]
> @xarlammm please go through https://github.com/thudugala/Plugin.LocalNotification/wiki/1.-Usage-10.0.0--.Net-MAUI

Hi Elvin, quick question: does the plugin work without MAUI? I've been developing iOS+Android natively and use only ex-Xamarin.Essentials parts of Maui namespaces, but it seems that having the core library is a must nowadays?

    Could not find `Microsoft.Maui` referenced by assembly `Plugin.LocalNotification, Version=11.1.3.0, Culture=neutral, PublicKeyToken=null`.
    Could not register the assembly 'Plugin.LocalNotification': System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.Maui, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies.
File name: 'Microsoft.Maui, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'

**Comment by thudugala** [comment]
> > @xarlammm please go through https://github.com/thudugala/Plugin.LocalNotification/wiki/1.-Usage-10.0.0--.Net-MAUI
> 
> Hi Elvin, quick question: does the plugin work without MAUI? I've been developing iOS+Android natively and use only ex-Xamarin.Essentials parts of Maui namespaces, but it seems that having the core library is a must nowadays?
> 
> ```
> Could not find `Microsoft.Maui` referenced by assembly `Plugin.LocalNotification, Version=11.1.3.0, Culture=neutral, PublicKeyToken=null`.
> Could not register the assembly 'Plugin.LocalNotification': System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.Maui, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies.
> ```

Yes, look at how the plugin is set natively in Xamarin. do, the same in MAUI native app
